.github/workflows/local-testnode/chain_config.json

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
How is this generated? It would be good to add a comment or script on how/when to update it in the future.

Reply:
This is the old state fuel-core was using by default before the latest release. So not generated by me I took it from their repo for CI.

Reply:
Maybe we could put this as a comment in `ci.yml` above the lines where it is used, so people know where to get the updated version.